### PR TITLE
Fixed bugs, style some

### PR DESCRIPTION
**Pull request by colev1**
-Fixed bugs that was displayed that map was undefined
-Update readme and add images
-Fixed some errors with the linter
-Still need to edit the search display area, how can we stop it from overflowing the page

### CODE REVIEW

**Comment by colev1** [comment]
🥇

File: src/Filter.js [hunk 1]
```diff
@@ -120,8 +118,8 @@ class Filter extends Component {
               return (
                 <li className="country-list">
                   {" "}
-                  <span className="country-name"> {country.name} </span>{" "}
-                  population: {country.population}{" "}
+                  <span className="country-name"> {country.name} </span>
```
Comment (Dhanciles) [inline src/Filter.js]: this is weird, i think the vs code linting added this

Comment (colev1) [reply src/Filter.js]: Agreed

Comment (colev1) [reply src/Filter.js]: @jamesb8891 is your linter acting wierd?

**Pull request merged**